server/app.ts

```diff
@@ -84,6 +84,36 @@ const getUserProgressData = async (req: Request) => {
   return progress
 }
 
+const getAccountData = async (req: Request, res: Response) => {
+  if (!req.user) {
+    return res.redirect('/signin')
+  }
+  if (req.user && !req.user.acceptedPolicies) {
+    return res.redirect('/eula')
+  }
```

Comment:
along the lines of what @Tansito mentioned, at this point we already know we have `req.user` (checked in statement above this one) so no need to check it here

```suggestion
  if (!req.user.acceptedPolicies) {
    return res.redirect('/eula')
  }
```